types/Enums.ts

```diff
@@ -81,3 +81,10 @@ export enum OrderPickupEventStatus {
   CANCELLED = 'CANCELLED',
   COMPLETED = 'COMPLETED',
 }
+
+export enum SocialMediaType {
+  LINKEDIN = 'LINKEDIN',
+  INSTAGRAM = 'INSTAGRAM',
+  FACEBOOK = 'FACEBOOK',
+  PORTFOLIO = 'PORTFOLIO',
+}
```

Comment:
Have we confirmed with membership/faris/etc. that these are the 4 social media types we are starting with?